### PR TITLE
add release notes

### PR DESCRIPTION
**Pull request by lvdbrink**
This PR adds release notes. I compiled the release notes by: 
- copying the major changes overview from the spec to the release notes intro
- going through all the closed pull requests and compiling the change table in clause 4, listing only the changes that affected the spec and leaving out editorial stuff
- compiling the substantive changes clause (6), describing each substantive change in the order they are mentioned in the major changes overview. 

Please check if: 
- all changes that should be mentioned are included 
- changes that were taken out again later are not included by mistake
- everything is described correctly

### CODE REVIEW

**Comment by situx** [comment]
Some remarks just browsing through your changes:
* The RIF rules were already part of GeoSPARQL 1.0. I believe we did not change anything there, except adapting them for the new literal types
* SPARQL Service Descriptions have been added, and all requirements of GeoSPARQL have been modeled in RDF
* SHACL shapes are also available as a test suite

File: 1.1/release_notes/sections/clause-1-intro.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
+== Introduction
+
+=== Scope
+
+The OGC GeoSPARQL standard supports representing and querying geospatial data on the Semantic Web. GeoSPARQL defines a vocabulary for representing geospatial data in RDF, and it defines extensions to the SPARQL query language for processing geospatial data. 
+
```
Comment (FransKnibbe) [inline 1.1/release_notes/sections/clause-1-intro.adoc:6]: I suggest changing "geospatial" to "spatial". There is not really an earth-based restriction in GeoSPARQL.

**Comment by FransKnibbe** [comment]
The release notes should probably be added to the profile declaration.

**Comment by lvdbrink** [comment]
> Some remarks just browsing through your changes:
> 
> * The RIF rules were already part of GeoSPARQL 1.0. I believe we did not change anything there, except adapting them for the new literal types

I copied the overview of changes in clause-1-intro from https://opengeospatial.github.io/ogc-geosparql/geosparql11/spec.html#_major_changes_between_versions_1_0_and_1_1. There, the RIF rules are listed as a new resource, and I guess they are, because with GeoSPARQL 1.0 they were not published as such. 

> * SPARQL Service Descriptions have been added, and all requirements of GeoSPARQL have been modeled in RDF
> * SHACL shapes are also available as a test suite

I will add those in a commit

> * Are the change notes a good place to link to the publications we did in the MDPI Journal etc. ?

Yes, good point, I will add a ref to the intro.

File: 1.1/release_notes/sections/clause-1-intro.adoc (hunk 1 above)
Comment (lvdbrink) [reply 1.1/release_notes/sections/clause-1-intro.adoc:6]: I copied this from the [GeoSPARQL 1.1 spec introduction](https://opengeospatial.github.io/ogc-geosparql/geosparql11/spec.html#_introduction). Wording it differently in the release notes might be confusing!

**Comment by lvdbrink** [comment]
> The release notes should probably be added to the profile declaration.

Yes, I could do that. Does anyone here know what the URI would be for the release notes?

**Comment by nicholascar** [comment]
Clarification on the RIF rules: in 1.0, there was a RIF template (in the Spec) and URI but no RIF resource. So we have actually created and added this in 1.1 (via a Python script and the template). I backfilled the RIF rules file into 1.0 too.

I created the RIF files with the 1.0 template and with this script, https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/rifgenerator.py, but I doubt if anyone's looked at them carefully, @situx, have you? As far as I can tell, RIF isn't used anywhere so we are really just maintaining the 1.0 legacy resource at this point. Having said that, I think we do need to have at least one review of the RIF files, if we can find someone to do this.  Review attempts given at #361

**Comment by situx** [comment]
@nicholascar I have not tested this set of RIF rules. All we have done is to write a compliance test that would test for RIF rule results to be correct. To be honest I have not used RIF rules much in practice, so it is good that you ask the people involved to review what we have done.

**Comment by nicholascar** [comment]
I have dealt with RIF concerns in two issues:

* #361 
* #366

**Review by nicholascar — APPROVED** [review]

**Review by nicholascar — APPROVED** [review]

File: 1.1/release_notes/sections/clause-1-intro.adoc (hunk 1 above)
Comment (jabhay) [reply 1.1/release_notes/sections/clause-1-intro.adoc:6]: I'd agree. We either need to change it in both places, or leave as is.

**Comment by nicholascar** [comment]
> profile

I've done this now!

**Review by jabhay — APPROVED** [review]
Looks like a good start. We're keen to move this through to amend once we complete the actual editorial work.

**Comment by jabhay** [comment]
Fixes https://github.com/opengeospatial/ogc-geosparql/issues/170

**Pull request merged**